js/views/cfi_navigation_logic.js

```diff
@@ -187,6 +187,28 @@ var CfiNavigationLogic = function(options) {
 
     /**
      * @private
+     * Helper function because of #386, node.parentElement returns null for text nodes on IE
+     * 
+     * @returns { node } parentElement
+     */
+    function getParentElement(node) {
+        if (node.parentElement) {
+            return node.parentElement;
```

Comment:
Note to self: `node.parentElement` !== `textRange.parentElement()` (the latter is a function).